mono/mini/mini.h

```diff
@@ -348,6 +348,7 @@ typedef struct MonoBasicBlock MonoBasicBlock;
 typedef struct MonoSpillInfo MonoSpillInfo;
 
 extern MonoCallSpec *mono_jit_trace_calls;
+extern MonoMethodDesc *mono_stats_method_desc;
```

Comment:
What's the thought process behind moving the header the method name lives in?

Reply:
I didn't really think about it. I just put it in the same file as `mono_jit_compile_method_inner`, which is where we use it. I can put it back in `mini-runtime.h`